scout/server/blueprints/cases/templates/cases/case.html

```diff
@@ -207,20 +205,52 @@ <h4 class="card-header">Case: {{case.display_name}}</h4>
   </div>
 {% endmacro %}
 
-{% macro roh_panel() %}
-  {% if case.chromograph_prefixes %}
-  <div class="d-flex p-2 ">
-    <div class="card panel-default" id="rohPanel">
-      <div class="panel-heading">Ideogram, UPD, ROH</div>
-      <br>
-      <div class="panel-body">
-        <svg id="roh_svg" width="1500" height="1250"></svg>
+
+<!-- Iterate through Individuals, if chromograph_files is presenet add a panel for images -->
+{% macro insert_multi_image_panel() %}
+  {% for i in case.individuals %}
+  {% if i.chromograph_images %}
+  {{ add_collapsable_panel(i.individual_id, i.display_name) }}
+  {% else %}
+    <a> {{i.individual_id}} images not configured</a>
+      {% endif  %}
+  {% endfor %}
+{% endmacro %}
+
+
+<!-- Create a collapsing panel. Used for -->
+
+{% macro add_collapsable_panel(id_tag, display_name) %}
+  <div class="row">
+    <div class="col-md-12">
+      <div id={{"collapse_" + id_tag}}>
+	<div class="card panel-default">
+	  <div class="panel-heading" >
+            <div class="panel-heading" href={{"#collapsing_" + id_tag}} data-toggle="collapse" datatarget={{"#collapsing_" + id_tag}} aria-expanded="true" aria-controls={{"collapsing_" + id_tag}} data-parent={{"#collapse_" + id_tag}}> <h5 class="panel-heading">Images {{display_name}} &gt</h5>	<span class="pull-right clickable"><i class="glyphicon glyphicon-chevron-up"></i></span> </div>
+          </div>
+          <div id={{"collapsing_" + id_tag}} class="collapse" data-parent={{"#collapse_" + id_tag}} >
+            <div class="card-body">
+              <svg id={{"svg_"+id_tag}} width="1200" height="1700"> </svg>
```

Comment:
Det bråkar med minskande av fönsterstorleken också: vet att vi löste det sist. Minns inte exakt hur, men kan kolla tillbaka på tidigare PR vid behov.

Reply:
Ok. I'll revisit the window layout.

Reply:
Fix pushed! Try it on you monitor and let me know how it works.

Reply:
Yeah, solves the between two and three columns transition, but still wont work for less than two. Just folding at that point would be acceptable, but it is a little odd as it is now.